app/pods/publications/publication/controller.js

```diff
@@ -113,10 +113,30 @@ export default class PublicationController extends Controller {
     this.model.save();
   }
 
+  translationDateBeforePublication(fieldname, event) {
+    const date = moment(new Date(event));
+    if (fieldname === 'publishBefore') {
+      const translateBefore = this.model.get('translateBefore');
+      return moment(translateBefore).isBefore(date, 'minutes');
+    }
+    if (fieldname === 'translateBefore') {
+      const publishBefore = this.model.get('publishBefore');
+      return moment(date).isBefore(publishBefore);
+    }
+  }
+
```

Comment:
Ik snap waarom deze methode er is, maar deze is eigenlijk niet erg herbruikbaar omdat enkel die constant shared is.
Het zou vb minder lijnen code zijn om lijn 117, 119 en 120 bij in setPublicationBeforeDate te steken en daar die .isBefore in een if te steken

Reply:
Ik had het geschreven om de test maar op een plaats te hebben nu moet.
Ik pas het aan.
indien we nu >= een > wordt  moeten we niet vergeten dit op 2 plaatsen aan te passen.